package-lock.json

```diff
@@ -1,8 +1,5391 @@
 {
   "name": "simple-icons",
+
+  "version": "5.4.0",
+  "lockfileVersion": 2,
```

Comment:
Can you revert all changes in this file, this project is still using `lockfileVersion` 1 rather than 2. It might be easiest to just copy the `package-lock.json` contents from the `develop` branch and copy them in here.